src/test/ssr.test.js

```diff
@@ -202,10 +202,8 @@ describe('ssr', () => {
     )
     const elements = sheet.getStyleElement()
 
-    expect(elements).toHaveLength(2)
-
+    expect(elements).toHaveLength(1)
```

Comment:
Why do we have fewer elements here? Shouldn't it still be one style tag for global styles + one style tag for local styles?

Reply:
There is no difference between local and global styles anymore (see commit msgs / PR description). SSR now produces a single style tag and it’s simply not worth or necessary to differentiate between local and global styles, especially since we’re getting rid of global styles soon.